SU-SWS/jumpstart_ui
# Add custom page title block.

# READY FOR REVIEW

# Summary
- Creates an easy way to put an H1 on a page where a page title is not available (or not working) in layout builder.

# Review By (Date)
- May 6th

# Urgency
- Normal

# Steps to Test

1. Check out this branch
2. Clear cache
3. Navigate to a layout builder page either on a node or on the page manager page
4. Add a new block and select "Custom Heading Block" under Jumpstart UI. 
5. Add some text for the page title and select a level of heading
6. Save it all and view on the front end for the expected results

# Affected Projects or Products
- D8CORE
- SOE
- Stanford Person Accelerant

# Associated Issues and/or People
- D8CORE-1632
- https://github.com/SU-SWS/stanford_person/pull/82
